src/__tests__/FormField.js

```diff
@@ -0,0 +1,350 @@
+import React from 'react'
```

Comment:
Shouldn't this be showing the diff as a `git mv`? Did you remove the `__tests__/InputField.js` file?